addons/sale/models/ir_config_parameter.py

```diff
@@ -1,31 +1,40 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import models, api
+from odoo import api, models
 from odoo.tools.misc import str2bool
 
 
 class IrConfigParameter(models.Model):
     _inherit = 'ir.config_parameter'
 
-    def _sale_sync_cron(self, unlink=False):
-        for config in self:
-            if (
-                config.key == 'sale.automatic_invoice'
-                and (send_invoice_cron := self.env.ref('sale.send_invoice_cron', raise_if_not_found=False))
-            ):
-                send_invoice_cron.active = False if unlink else str2bool(config.value)
+    # Mapping between config parameters and the Sales-related crons they toggle.
+    _sale_param_cron_mapping = {
+        'sale.async_emails': 'sale.send_pending_emails_cron',
+        'sale.automatic_invoice': 'sale.send_invoice_cron',
+    }
```

Comment:
It's not the usual 'odoo' way to use class variables like that, why can't it be a constant above the class instead (only for the tests ?)

Reply:
why not a getter method then? :thinking:

Reply:
It's more code to achieve the same result but I can add one if you prefer.

Currently it works like this to extend
```python
_sale_param_cron_mapping = {
    **super()._sale_param_cron_mapping,
    'new_key': 'new_value',
)